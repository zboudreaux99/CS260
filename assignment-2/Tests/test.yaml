test:
  timelimit: 10
  launch: ./tally
  params: '@T/albums.csv @T/requests.csv'
  stderr: false
dueDate: 12/06/2024
latePenalties: [10, 20, 100]
build:
  command: make
  weight: 1
  studentFiles:
    include: ["playlist.h", "playlist.cpp"]
  instructorFiles:
    exclude: ["playlist.h", "playlist.cpp"]
