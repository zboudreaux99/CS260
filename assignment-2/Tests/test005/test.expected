Read 1 songs.
Processed 4 playlist items.

Look Back In Anger       David Bowie         David Bowie - Live on Ma 1 03:05
Total: 03:05

